src/main/webapp/script.js

```diff
@@ -105,13 +105,16 @@ function numberPage(pageNumber) {  // eslint-disable-line
   currentPage = pageNumber;
   addThreads();
 }
+
+/** Check if previous and next buttons can go onto next or not */
```

Comment:
nit: "Dis/enables previous and next buttons". Technically, since it's modifying the buttons instead of just reading the state, it would be more readable to call it #update().